### ISSUE TITLE
Does the splash screen img need alt text?

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#490]
I ran an a11y audit on a https://phet-dev.colorado.edu/html/resistance-in-a-wire/1.5.0-rc.3/phet/resistance-in-a-wire_en_phet.html, and got the following critique:
![image](https://user-images.githubusercontent.com/6856943/39601572-d39d0780-4ece-11e8-89ad-f8db37543024.png)

It doesn't stick around for long, but I think we should still have it. Tagging @terracoda for a11y context and @samreid because he wrote the code for thoughts.

**Comment by samreid** [comment]
As far as I know, the text for the startup sequence has been highly refined, and I wonder if having alt text for the splash image would disturb that.  On the other hand, if a user sees there is an image with no alt-text, they may be dismayed or confused.  Not sure what's best here.

**Comment by zepumph** [comment]
>As far as I know, the text for the startup sequence has been highly refined

What do you mean by this?

**Comment by samreid** [comment]
It is my understanding that the accessibility design team had already decided what the simulation should say as it starts up, not sure if adding an alt text would interfere with that.  For example, when you start balloons and static electricity, it says:

"Balloons and Static Electricity Scene Summary.  This is an interactive sim.  It changes as you play with it.  The sim has a Play Area and Control Panel...."

It may be an inferior user experience if it instead said:

"The PhET Simulation Logo.  Balloons and Static Electricity Scene Summary. This is an interactive sim.  It changes as you play with it.  The sim has a Play Area and Control Panel..."

**Comment by samreid** [comment]
MDN suggests using empty alt text for cases where the image is decoration only:
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#Empty_alt_attributes

> The reason to use an empty alt instead of not including it is because many screen readers announce the whole image URL if no alt is provided.  In the above example, the image is acting as a visual decoration to the heading its associated with. In cases like this, and in cases where an image is only decoration and has no content value, you should put an empty alt on your images. Another alternative is to use the aria role attribute role="presentation" — this also stops screens readers from reading out alternative text.

**Comment by terracoda** [comment]
The image element requires an alt attribute. The alt attribute can be null if the image does not have any relevant important information.

```html
<img src="path/name_of_image.jpg" alt="">
```
If there is no alt at all, the browser will use the file name as default.

**Comment by terracoda** [comment]
Regarding https://github.com/phetsims/joist/issues/490#issuecomment-386430377, there is no text for the actual loading splash page (i.e., no text for the PhET Logo). There will be a sound for that indicates that the sim is loading.

The "highly refined" text starts at the beginning of the actual sim page, either the opening page on a single screen sim, or the content of the home screen on a multi-screen sim.

If PhET needs to be identified for every simulation, we could use the page title of the simulation. For example, here a re few options:
```html
<title>Balloons and Static Electricity - PhET Interactive Simulations</title>
<title>PhET Interactive Simulations - Balloons and Static Electricity</title>
<title>Balloons and Static Electricity, a PhET Simulation</title>
<title>Balloons and Static Electricity, a PhET Sim</title>
```
The purpose of the page `title` is to reassure the student that they are indeed in the right place.

**Comment by zepumph** [comment]
Fixed,  tested and audited in https://bayes.colorado.edu/dev/html/faradays-law/1.4.0-dev.26/phet/faradays-law_en_phet.html. @samreid please review.

**Comment by samreid** [comment]
@zepumph, thanks for your fix. The change looks good and I tested that the alt appears in the splash image on startup in requirejs.  To simplify the readability of future commits, I would recommend doing a prior "reformat" commit before the salient change.  I think you normally do this, just wanted to remind you for next time.  Closing.
